packages/graphiql/src/components/__tests__/GraphiQL.spec.tsx

```diff
@@ -114,9 +114,11 @@ describe('GraphiQL', () => {
   });
 
   it('defaults to the built-in default query', async () => {
```

Comment:
Fixed up this test which could never fail (`findByText` returns a promise which is always truthy).